src/parser.ts

```diff
@@ -355,10 +355,20 @@ const EXPR_BUILDER: MarkVisitor<NodeTypes.ExprNode> = {
       return result
     }
 
-    const args: NodeTypes.ExprNode[] = []
+    let args: NodeTypes.ExprNode[] = []
```

Comment:
No reason to change to `let` here? 🤔

Reply:
Yeah you're absolutely right, I pushed this up at the end of the day and didn't give it a proper review myself - a few messy things slipped through. That's my bad!